plugins/rbac/src/components/RolesList.tsx

```diff
@@ -30,6 +30,12 @@ export const RolesList = () => {
   const classes = useStyles();
   const { loading, data, retry, createRoleAllowed } = useRoles();
 
+  React.useEffect(() => {
+    return () => {
+      setToastMessage('');
+    };
+  }, [setToastMessage]);
```

Comment:
this doesn't seem right, toast ideally should be across the app if there is something at App level or at at plugin level.

This we can take as follow issue if needed

Reply:
Will look into this as part of a separate ticket